x-pack/plugins/observability/public/pages/alerts/components/alerts_search_bar.tsx

```diff
@@ -54,9 +54,6 @@ export function AlertsSearchBar({
       timeHistory={timeHistory}
       dateRangeFrom={rangeFrom}
       dateRangeTo={rangeTo}
-      onRefresh={({ dateRange }) => {
```

Comment:
`onRefresh` was not working as expected, when clicking on the refresh button, `onQuerySubmit` was called, I assume it is related to how this `SearchBar` is configured.

Let me know if someone has an idea about it.